sirepo/package_data/static/js/sirepo.js

```diff
@@ -414,35 +414,28 @@ SIREPO.app.factory('appState', function(errorService, fileManager, requestQueue,
                 return {
                     urlOrParams: 'saveSimulationData',
                     successCallback: function (resp) {
-                        if (resp.error && resp.error == 'invalidSerial') {
-                            srlog(
-                                resp.simulationData.models.simulation.simulationId,
-                                ': update collision newSerial=',
-                                resp.simulationData.models.simulation.simulationSerial,
-                                '; refreshing'
-                            );
-                            refreshSimulationData(resp.simulationData);
-                            errorService.alertText(
-                                "Another browser updated this simulation."
-                                + " This window's state has been refreshed."
-                                + " Please retry your action."
-                            );
+                        if (resp.error) {
+                            if (resp.error === 'invalidSerial') {
```

Comment:
I fixed the code in #6300 to remove this test here, and instead to [refresh the app](https://github.com/radiasoft/sirepo/pull/6300/files#diff-c1af31ef028c0a9ee6c6e71acb4b0ad646d4136dd06d3a1bc9cdb11c55fbc801R588). It never happens, and when it does, a complete refresh is probably safest given the amount of caching in the app and the fact that simulationData was never used.

Your new code (after #6300 is merged shortly) should simply remove the errorCallback. Easiest thing would be revert this change in your branch so you don't get a conflict.